File: challenges/project-euler/python3/6.py
def sum_of_squares(count):
    total = 0
    for i in range(1, count + 1):
        total += i ** 2
    return total

def square_of_sums(count):
    total = 0
    for i in range(1, count + 1):
        total += i
    return total ** 2

print(square_of_sums(100) - sum_of_squares(100))

File: challenges/project-euler/c/6.c
#include <stdio.h>

int main()
{
	int sum_of_squares = 0, sum = 0;
	for (int i = 1; i <= 100; i++)
	{
		sum += i;
		sum_of_squares += (i * i);
	}
	int square_of_sums = sum * sum;
	printf("Sum of squares: %d\n\rSquare of sums: %d\n\rDifference: %d\n\r", sum_of_squares, square_of_sums, square_of_sums - sum_of_squares);
	return 0;
}
